app/controllers/Application.scala

```diff
@@ -365,6 +355,7 @@ object Application extends Controller with Security {
   }
 
   def scheme(id: Int) = isAnalyst { identity => implicit request =>
+    // todo: update to currentSubscriberId
```

Comment:
I noticed that as well just now. I think until you added Interests 2.0 to that page it seemed like just a page with the Scheme name and link to adminy things so the `isAnalyst` restriction made sense to me. With that being the only entry point to add a new Interest, we do indeed need a ticket to update the required role.